lua/nordic/tests/init.lua

```diff
@@ -1,13 +1,10 @@
-local U = require('nordic.utils')
-
 M = {}
 
 function M.assert_eq(left, right, message)
```

Comment:
I think we should also move this to utils.

Reply:
you mean `assert_eq`? If so, could you explain why I don't imagine using it anywhere except in tests?

Reply:
Just semantics, I like having utils stuff separated.  Not that big of a deal though.

Reply:
I am not sure I like that, but it's scope is technically not expanding: [refactor(tests): move assert_eq to utils;](https://github.com/AlexvZyl/nordic.nvim/pull/150/commits/a6a9953d40a35244445286eec299c37ea4ff4d18)

Reply:
You can move it back if you want.